mods/d2k/rules/structures.yaml

```diff
@@ -593,7 +593,7 @@ large_gun_turret:
 		QuantizedFacings: 32
 	Armament:
 		Weapon: TowerMissile
-		LocalOffset: 256,384,768, 256,-384,768
+		LocalOffset: 256,0,768
```

Comment:
Applies to missile tank as well.

We could also leave damage as it is, but still use Burst: 2 and make the BurstDelay identical to ReloadDelay, then the rate of fire would remain the same but it would alternate between left and right.